test/asv.linux.conf.json

```diff
@@ -34,20 +34,20 @@
         "python -c \"import shutil; shutil.rmtree('{build_dir}/qiskit', True)\"",
         "python -c \"import shutil; shutil.rmtree('{build_dir}/qiskit_aer.egg-info', True)\"",
         "pip install git+https://github.com/Qiskit/qiskit-terra",
-        "pip install git+https://github.com/Qiskit/qiskit-aqua",
+        "pip install qiskit_nature",
         "python -mpip install {wheel_file}"
     ],
     "uninstall_command": [
         "return-code=any python -mpip uninstall -y qiskit-terra",
-        "return-code=any python -mpip uninstall -y qiskit-aqua",
+        "pip uninstall qiskit_nature",
```

Comment:
maybe add the `-y` here to not fail if it's not installed for some reason. Also maybe instead of doing this we can use the `matrix` kwarg and fix the version: https://asv.readthedocs.io/en/stable/asv.conf.json.html?highlight=matrix#matrix